### ISSUE TITLE
"Skip, but then..."

### ISSUE THREAD

**Issue by howardjones** [sampsyo/beets#1255]
I seem to regularly want to tell beets to skip the import of an album, but also put the skipped album in another folder, or tag it in some way. e.g. "Skip, and move the the duplicates folder" or "Skip, and move out of the way while I figure out what is up with the file names".

I don't see anything in the plugin documentation about adding options to the import command list. Would this kind of feature require changes in the core? (and do you want it? :-) )

**Comment by sampsyo** [comment]
Hi! Yes, I can imagine stuff like this being useful. Currently, if you were an enterprising plugin hacker, you could do something *similar* by just listening for one of the import task events (e.g., `import_task_files`?) and acting on skipped albums. But that wouldn't give you an interactive choice.

A plugin hook to add additional interactive choices would certainly be interesting, and it has come up before (I don't have an example at hand right now, but trust me here :smiley:). It would need to be done extremely carefully to avoid confusing prompts in the UI, but I invite you to explore the idea if you're interested. Let me know if you're interested and we can talk more specifics.

**Comment by redinkinc** [comment]
Hi @sampsyo. I'm new in using beets and currently merging my music folders. I experienced the same issue of having duplicates/albums with missing tracks/unknown albums aso. and having no clue after import which category the leftover tracks are. Are you currently developing new functionalities or plugins dealing with this? I can also offer some help, if needed.
I also recognized that beets doesn't notice duplictes when imported as "use-as-is" when the album is unknown. Is it possible to search for use-as-is duplicates?
Thanks, Fabian
